### PR TITLE
solution

### PR DESCRIPTION
**Pull request by tiserett**
[DEMO LINK](https://tiserett.github.io/react_article/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -8,37 +9,15 @@ export const firstArticle = {
...
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime
-          assumenda magni adipisci esse quidem ratione ipsa labore vero illo
-          optio.
-        </p>
-      </div>
+      <Article
+        title={firstArticle.title}
+        date={firstArticle.date}
+        text={firstArticle.text}
+      />
```
Comment (sergii-nosachenko) [inline src/App.jsx:20]:
```suggestion
      <Article {...firstArticle} />
```

'Destructurize' battle begins!..)

**Review by sergii-nosachenko — APPROVED** [review]
Excellent 🚀... almost 😁

**Review by sergii-nosachenko — APPROVED** [review]
🔥

File: src/components/Article.jsx [hunk 2]
```diff
@@ -1 +1,39 @@
...
+      </p>
+    </div>
+  );
+};
+
+Article.propTypes = {
+  title: PropTypes.string.isRequired,
+  date: PropTypes.string.isRequired,
+  text: PropTypes.string.isRequired,
+};
```
Comment (AlinaMatuschak) [inline src/components/Article.jsx:39]: 🔥

**Review by AlinaMatuschak — APPROVED** [review]

**Pull request closed without merging**